silkworm/silkrpc/http/request_handler.cpp

```diff
@@ -107,7 +107,7 @@ bool RequestHandler::is_valid_jsonrpc(const nlohmann::json& request_json) {
         SILK_TRACE << property_name << " : " << property.value().type_name() << " : " << property.value().dump();
 
         if (property_name == "id") {
-            if (!property.value().is_number()) {
+            if (!property.value().is_number() && !property.value().is_string()) {
```

Comment:
Strictly speaking, we should support also `NULL` value for incoming `id` field according to the [spec](https://www.jsonrpc.org/specification#id1)